### PR TITLE
doxygen: 1.8.20 -> 1.9.1

### PR DESCRIPTION
**Pull request by bhipple**
###### Motivation for this change

Unfortunately, the cmake files are not parallel safe (even on the current
version), so this also disables parallel building.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/ripgrep/default.nix [hunk 1]
```diff
@@ -10,7 +10,7 @@
 }:
 
 rustPlatform.buildRustPackage rec {
-  pname = "ripgrep";
```
Comment (ajs124) [inline pkgs/tools/text/ripgrep/default.nix:13]: Seems unrelated

Comment (bhipple) [reply pkgs/tools/text/ripgrep/default.nix]: Oops, indeed; ty.

**Comment by bhipple** [comment]
Rebased onto staging due to number of rebuilds; sry it seems GitHub has CC'd a whole bunch of people ...

**Comment by SuperSandro2000** [comment]
> Rebased onto staging due to number of rebuilds; sry it seems GitHub has CC'd a whole bunch of people ...

If you use the rebase action that will not happen. Sadly there is no other easy way to avoid this that I am aware of right now.

**Comment by ajs124** [comment]
Assuming there are no conflicts you can also rebase onto the merge-base of master and staging, force push that and retarget afterwards. That also shouldn't cause any pings.

**Comment by SuperSandro2000** [comment]
python3.8-pandas segfaults with this update

```
tests/io/pytables/test_store.py .........................Fatal Python error: Segmentation fault
```
